samples/sample_calibrate_eye_to_hand.py

```diff
@@ -0,0 +1,77 @@
+"""Hand-eye calibration sample."""
+import datetime
+
+import numpy as np
+import zivid.handeye
+
+
+def _acquire_checkerboard_frame(camera):
+    print("Capturing checkerboard image... ")
+    with camera.update_settings() as updater:
+        updater.settings.iris = 17
+        updater.settings.gain = 1.0
+        updater.settings.exposure_time = datetime.timedelta(microseconds=20000)
+        updater.settings.filters.gaussian.enabled = True
+    print("OK")
+    return camera.capture()
+
+
+def _enter_robot_pose(index):
+    inputted = input(
+        "Enter pose with id={} (a line with 16 space separated values describing 4x4 row-major matrix):".format(
+            index
+        )
+    )
+    elements = inputted.split(maxsplit=15)
+    data = np.array(elements, dtype=np.float64).reshape((4, 4)).transpose()
```

Comment:
The user shouldn't have to transpose the numpy matrix before passing it into our API. We should rather make the wrapper so that it just works, given a matrix that is correct in numpy.

I'll look into this now.

Reply:
Ah yes, Eigen defaults to column-major storage, while we want row-major (both the numpy matrix and our internal matrix is row-major).

I'll put the necessary changes in comments on the appropriate lines.